themes/finna2/templates/RecordDriver/DefaultRecord/audio-information.phtml

```diff
@@ -1,7 +1,7 @@
 <!-- START of: finna - RecordDriver/DefaultRecord/audio-information.phtml -->
 <?php
     if ($fileSize = $url['data']['size'] ?? '') {
-        $fileSize = !empty($fileSize['value']) && $fileSize['unit'] === 'byte' ? '(' . $this->fileSize(preg_replace('/[^0-9]/', '', $fileSize['value'])) . ')' : '';
+        $fileSize = !empty($fileSize['value']) ? '(' . $this->fileSize(preg_replace('/[^0-9]/', '', $fileSize['value'])) . ')' : '';
```

Comment:
Eihän tämä kuitenkaan näin voi mennä. Jos unit on jotain muuta kuin byte, niin ei fileSize anna sille oikeaa arvoa. Toisin sanoen pitää joko konvertoida tavuiksi SolrLido:ssa, välittää myös yksikkö fileSize:lle tai näyttää koko jotenkin vaihtoehtoisesti.

Reply:
Lisäksi pitää huomioida, että muu kuin tavu voi olla desimaaliluku, vaikka "3.5", jolloin ei voi suoraan vain poistaa ei-numeroita stringistä. Muuten olisi kyllä parempi tehdä siivous SolrLidon puolella, jotta sitä ei tarvitse erikseen tehdä monessa paikassa templaateissa.